### PR TITLE
CloudMigrations: point cloud instance to sef-serve guide

### PR DESCRIPTION
**Pull request by macabu**
Fixes https://github.com/grafana/grafana-operator-experience-squad/issues/939

The app and link is available on hosted Grafana instances.

### CODE REVIEW

File: public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const MigrationStepsPane = () => {
       <TextLink href="/connections/private-data-source-connections">
         {t('migrate-to-cloud.get-started.configure-pdc-link', 'Configure PDC for this stack')}
       </TextLink>
-      <TextLink href="https://grafana.com/docs/grafana-cloud/account-management/migration-guide">
+      <TextLink href="/a/grafana-selfservetutorials-app/migrate-oss">
```
Comment (danielkenlee) [inline public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx:57]: I think the first `/` may not be needed @macabu, but please test first!

Comment (danielkenlee) [reply public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx:57]:
```suggestion
<TextLink href="a/grafana-selfservetutorials-app/migrate-oss">
```

**Review by danielkenlee — CHANGES_REQUESTED** [review]

File: public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx (hunk 1 above)
Comment (macabu) [reply public/app/features/migrate-to-cloud/cloud/EmptyState/MigrationStepsPane.tsx:57]: @danielkenlee It works as-is 🙂

**Review by leandro-deveikis — APPROVED** [review]

**Review by ishanjainn — APPROVED** [review]

**Review by danielkenlee — APPROVED** [review]

**Pull request merged**